hie-base/Haskell/Ide/Engine/PluginTypes.hs

```diff
@@ -175,8 +179,51 @@ type PluginName = T.Text
 data IdePlugins = IdePlugins
   { ipMap :: Map.Map PluginId [UntaggedCommandDescriptor]
   } deriving (Show,Eq,Generic)
+instance ToSchema IdePlugins
+
+-- ---------------------------------------------------------------------
+
+-- | A position in a source file
+-- type Pos = (Line,Col)
+data Pos = Pos { line::Line, col:: Col} deriving (Generic,Show,Read,Eq,Ord)
+-- NOTE: Pos needs to be defined in record syntac otherwise the generically
+--       derived swagger schema does not work properly
+instance ToSchema Pos
+
+unPos :: Pos -> (Int,Int)
+unPos (Pos (Line l) (Col c)) = (l,c)
+
+toPos :: (Int,Int) -> Pos
```

Comment:
Yes, it was a quick and dirty fix to make updating the tests easier. 

So no, not really, I guess.